### ISSUE TITLE
CI issues

### ISSUE THREAD

**Issue by bartolkaruza** [react-native-community/react-native-cameraroll#34]
[Android build](https://circleci.com/gh/react-native-community/react-native-cameraroll/150) is failing on master with the following error:
```
> Task :app:compileDebugAndroidTestJavaWithJavac FAILED
/root/project/example/android/app/src/androidTest/java/com/camerarollexample/DetoxTest.java:22: error: incompatible types: android.support.test.rule.ActivityTestRule<MainActivity> cannot be converted to androidx.test.rule.ActivityTestRule
        Detox.runTests(mActivityRule);
                       ^
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
```

[iOS build] is failing on master with the following error:
```

the transform cache was reset.

❌  error: File /Users/distiller/project/example/ios/build/Build/Products/Release-iphonesimulator/CameraRollExample.app/main.jsbundle does not exist. This must be a bug with


** BUILD FAILED **
```

**Comment by bartolkaruza** [comment]
#38 fixes the android issue

**Comment by bartolkaruza** [comment]
temporarily resolved by #40 (by turning off the native builds). We should look at resolving the iOS issue still as part of re-enabling e2e on CI.
